### PR TITLE
iOS Safari crash

### PR DESCRIPTION
**Pull request by stephencookdev**
# Problem
Currently iOS Safari crashes when repeatedly entering/exiting the Selfie page (e.g. take a selfie, and then "redo" a lot)

# Solution
A change was made to react-webcam-onfido (see https://github.com/onfido/react-webcam/pull/29), which this PR leverages, to sidestep the issue.

## Checklist
- [X] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [X] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,7 +111,7 @@
     "react-native-listener": "1.0.1",
     "react-phone-number-input": "^0.15.2",
     "react-redux": "4.4.6",
-    "react-webcam-onfido": "^0.1.10",
+    "react-webcam-onfido": "git://github.com/onfido/react-webcam.git#feature/fix-ios-safari",
```
Comment (stephencookdev) [inline package.json]: Looks like the `flow` and npm config in this repo are set up in such a way that this doesn't work :/

So you'll have to build this locally to preview the change...

But yes, all the actual _code_ to review is [in this PR](https://github.com/onfido/react-webcam/pull/29)

Comment (stefaniacardenas) [reply package.json]: Feel free to publish `react-webcam-onfido` when it's approved and the acceptance can be done via this PR

**Comment by stephencookdev** [comment]
I've published the react-webcam-onfido as a `rc1` version, so you should be able to actually test the changes through this PR now

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-564-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://564-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**